azurerm/resource_arm_app_service_virtual_network_connection_test.go

```diff
@@ -0,0 +1,115 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccAzureRMAppServiceVirtualNetworkConnection_basic(t *testing.T) {
+	resourceName := "azurerm_app_service_virtual_network_connection.example"
+	ri := tf.AccRandTimeInt()
+	config := testAccAzureRMAppServiceVirtualNetworkConnection_basic(ri, testLocation())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMAppServiceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAppServiceExists(resourceName),
+					resource.TestCheckResourceAttrSet(resourceName, "name"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrSet(resourceName, "type"),
+					resource.TestCheckResourceAttrSet(resourceName, "certificate_blob"),
+					resource.TestCheckResourceAttrSet(resourceName, "certificate_thumbprint"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccAzureRMAppServiceVirtualNetworkConnection_basic(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "example" {
+	name     = "acctestRG-%d"
```

Comment:
could we use 
```suggestion
	name     = "acctestRG-appservice-%d"
```

here please?